dashboard/package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "dashboard",
+  "proxy": "https://localhost",
```

Comment:
Just for my understanding, what is the purpose of adding the proxy here? I don't mean [technically](https://create-react-app.dev/docs/proxying-api-requests-in-development/), but more the reasoning behind. Thanks! 😄

Reply:
Hahaha, it is just me clicking the wrong "commit all" button instead of "commit staged" one :facepalm: . Thanks for pointing this out.

Reply:
Great!
If it helps, in order to avoid having to modify the `package.json` for this purpose, I add in my local a `src/setupProxy.js` file like this
```
const createProxyMiddleware = require('http-proxy-middleware');

module.exports = function(app) {
  app.use(
    ['/apis', '/api'],
    createProxyMiddleware({
      target: 'http://localhost',
      changeOrigin: true,
    })
  );
};
```
It is also a file that could be committed by mistake, but maybe we could add it to `.gitignore` 😅